[🐛 Bug]: Port ____ already In use with selenium/node-chrome:4.1.0-20211123

### What happened?

I was getting an error message ```Unable to execute request for an existing session: Unable to find session with ID:``` and saw a possible solution described in https://github.com/SeleniumHQ/selenium/issues/9988 and followed a hint in the ticket to try running the 4.1 prerelease.

when staring the hub and nodes I get errors similar to this for each chrome node:

```java.io.IOException: Port 5502 already in use```

I am running selenium grid under podman 3.3.1, which I know is not supported. The startup command podman-compose is the podman equivalent to docker-compose

Is there a difference in how ports are defined or specified with the prerelease, either in docker-selenium or in selenium proper?
### Command used to start Selenium Grid with Docker

```shell
podman-compose --project-name grid -f container-compose.yml up --build --remove-orphans -d
```

file container-compose.yml: (updated 11/23 with simplified version)
```
services:
    chrome1:
        image: selenium/node-chrome:4.1.0-20211123
        privileged: true
        hostname: chrome1
        shm_size: 2G
        environment:
            - VNC_NO_PASSWORD=1
            - VNC_PORT=5901
            - NO_VNC_PORT=7901
            - VNC_VIEW_ONLY=1
            - SE_NODE_GRID_URL=http://selgrid:4444
            - SCREEN_WIDTH=1550
            - SCREEN_HEIGHT=880
            - SCREEN_DPI=72
            - SCREEN_DEPTH=16
            - START_NO_VNC=true
            - DISPLAY=:91
            - DISPLAY_NUM=91
            - SE_OPTS=--log-level FINE --registration-secret xt3jrtsQW3KFK9em4bCy --host selgrid -p 5501
            - SE_EVENT_BUS_HOST=selenium-hub
            - SE_EVENT_BUS_PUBLISH_PORT=4442
            - SE_EVENT_BUS_SUBSCRIBE_PORT=4443
            - SE_SESSION_REQUEST_TIMEOUT=500
            - SE_SESSION_RETRY_INTERVAL=2
            - CONTAINER_HOST=tcp://selgrid:23750
            - TZ=US/Eastern
            - DOCKER_HOST=tcp://selgrid:23750
            - JAVA_OPTS=-Dio.netty.tryReflectionSetAccessible=true --add-opens java.base/jdk.internal.misc=ALL-UNNAMED
        ports:
            - "5901:5901"
            - "7901:7901"
            - "5501:5501"

    selenium-hub:
        image: selenium/hub:4.1.0-20211123
        privileged: true
        hostname: selenium-hub
        labels:
            - org.broadinstitute.grid.service=hub
        environment:
            - SE_SESSION_REQUEST_TIMEOUT=500
            - SE_SESSION_RETRY_INTERVAL=2
            - SE_OPTS=--log-level WARNING --registration-secret xt3jrtsQW3KFK9em4bCy
            - VNC_VIEW_ONLY=1
            - CONTAINER_HOST=tcp://selgrid:23750
            - DOCKER_HOST=tcp://selgrid:23750
            - JAVA_OPTS=-Dio.netty.tryReflectionSetAccessible=true --add-opens java.base/jdk.internal.misc=ALL-UNNAMED
            - TZ=US/Eastern
        ports:
            - 4444:4444

```


### Relevant log output

```shell
Starting Selenium Grid Node...
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by io.netty.util.internal.ReflectionUtil (file:/opt/selenium/selenium-server.jar) to constructor java.nio.DirectByteBuffer(long,int)
WARNING: Please consider reporting this to the maintainers of io.netty.util.internal.ReflectionUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.openqa.selenium.grid.Bootstrap.runMain(Bootstrap.java:77)
	at org.openqa.selenium.grid.Bootstrap.main(Bootstrap.java:70)
Caused by: java.io.UncheckedIOException: java.io.IOException: Port 5502 already in use
	at org.openqa.selenium.netty.server.NettyServer.start(NettyServer.java:159)
	at org.openqa.selenium.grid.node.httpd.NodeServer$1.start(NodeServer.java:192)
	at org.openqa.selenium.grid.node.httpd.NodeServer$1.start(NodeServer.java:189)
	at org.openqa.selenium.grid.node.httpd.NodeServer.execute(NodeServer.java:231)
	at org.openqa.selenium.grid.TemplateGridCommand.lambda$configure$4(TemplateGridCommand.java:129)
	at org.openqa.selenium.grid.Main.launch(Main.java:83)
	at org.openqa.selenium.grid.Main.go(Main.java:57)
	at org.openqa.selenium.grid.Main.main(Main.java:42)
	... 6 more
Caused by: java.io.IOException: Port 5502 already in use
	... 14 more
Caused by: java.net.BindException: Cannot assign requested address
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:455)
	at java.base/sun.nio.ch.Net.bind(Net.java:447)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:134)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:562)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334)
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506)
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491)
	at io.netty.handler.logging.LoggingHandler.bind(LoggingHandler.java:230)
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506)
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491)
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973)
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:260)
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:356)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
2021-11-17 13:08:58,219 INFO exited: selenium-node (exit status 1; not expected)
```


### Operating System

RHEL8

### Docker Selenium version (tag)

4.1.0-20211123

with log-level=FINE
```
Starting Selenium Grid Node...
10:57:12.668 INFO [LoggingOptions.configureLogEncoding] - Using the system default encoding
10:57:12.674 INFO [OpenTelemetryTracer.createTracer] - Using OpenTelemetry for tracing
10:57:12.763 INFO [UnboundZmqEventBus.<init>] - Connecting to tcp://selenium-hub:4442 and tcp://selenium-hub:4443
10:57:12.815 INFO [UnboundZmqEventBus.<init>] - Sockets created
10:57:13.817 INFO [UnboundZmqEventBus.<init>] - Event bus ready
10:57:13.944 INFO [NodeServer.createHandlers] - Reporting self as: http://selgrid:5501
10:57:13.963 INFO [NodeOptions.getSessionFactories] - Detected 12 available processors
10:57:14.021 INFO [NodeOptions.report] - Adding chrome for {"browserVersion": "95.0","browserName": "chrome","platformName": "Linux","se:vncEnabled": true} 1 times
10:57:14.030 INFO [Node.<init>] - Binding additional locator mechanisms: name, id, relative
10:57:14.102 DEBUG [MultithreadEventLoopGroup.<clinit>] - -Dio.netty.eventLoopThreads: 24
10:57:14.116 DEBUG [InternalThreadLocalMap.<clinit>] - -Dio.netty.threadLocalMap.stringBuilder.initialSize: 1024
10:57:14.116 DEBUG [InternalThreadLocalMap.<clinit>] - -Dio.netty.threadLocalMap.stringBuilder.maxSize: 4096
10:57:14.122 DEBUG [NioEventLoop.<clinit>] - -Dio.netty.noKeySetOptimization: false
10:57:14.122 DEBUG [NioEventLoop.<clinit>] - -Dio.netty.selectorAutoRebuildThreshold: 512
10:57:14.136 DEBUG [PlatformDependent0.explicitNoUnsafeCause0] - -Dio.netty.noUnsafe: false
10:57:14.137 DEBUG [PlatformDependent0.javaVersion0] - Java version: 11
10:57:14.139 DEBUG [PlatformDependent0.<clinit>] - sun.misc.Unsafe.theUnsafe: available
10:57:14.140 DEBUG [PlatformDependent0.<clinit>] - sun.misc.Unsafe.copyMemory: available
10:57:14.140 DEBUG [PlatformDependent0.<clinit>] - java.nio.Buffer.address: available
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by io.netty.util.internal.ReflectionUtil (file:/opt/selenium/selenium-server.jar) to constructor java.nio.DirectByteBuffer(long,int)
WARNING: Please consider reporting this to the maintainers of io.netty.util.internal.ReflectionUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
10:57:14.142 DEBUG [PlatformDependent0.<clinit>] - direct buffer constructor: available
10:57:14.142 DEBUG [PlatformDependent0.<clinit>] - java.nio.Bits.unaligned: available, true
10:57:14.144 DEBUG [PlatformDependent0.<clinit>] - jdk.internal.misc.Unsafe.allocateUninitializedArray(int): available
10:57:14.146 DEBUG [PlatformDependent0.<clinit>] - java.nio.DirectByteBuffer.<init>(long, int): available
10:57:14.146 DEBUG [PlatformDependent.unsafeUnavailabilityCause0] - sun.misc.Unsafe: available
10:57:14.148 DEBUG [PlatformDependent.maxDirectMemory0] - maxDirectMemory: 3087007744 bytes (maybe)
10:57:14.149 DEBUG [PlatformDependent.tmpdir0] - -Dio.netty.tmpdir: /tmp (java.io.tmpdir)
10:57:14.149 DEBUG [PlatformDependent.bitMode0] - -Dio.netty.bitMode: 64 (sun.arch.data.model)
10:57:14.150 DEBUG [PlatformDependent.<clinit>] - -Dio.netty.maxDirectMemory: 3087007744 bytes
10:57:14.150 DEBUG [PlatformDependent.<clinit>] - -Dio.netty.uninitializedArrayAllocationThreshold: 1024
10:57:14.151 DEBUG [CleanerJava9.<clinit>] - java.nio.ByteBuffer.cleaner(): available
10:57:14.151 DEBUG [PlatformDependent.<clinit>] - -Dio.netty.noPreferDirect: false
10:57:14.160 DEBUG [PlatformDependent$Mpsc.<clinit>] - org.jctools-core.MpscChunkedArrayQueue: available
10:57:14.194 DEBUG [DefaultChannelId.<clinit>] - -Dio.netty.processId: 97 (auto-detected)
10:57:14.196 DEBUG [NetUtil.<clinit>] - -Djava.net.preferIPv4Stack: false
10:57:14.197 DEBUG [NetUtil.<clinit>] - -Djava.net.preferIPv6Addresses: false
10:57:14.199 DEBUG [NetUtilInitializations.determineLoopback] - Loopback interface: lo (lo, 0:0:0:0:0:0:0:1%lo)
10:57:14.200 DEBUG [NetUtil$1.run] - Failed to get SOMAXCONN from sysctl and file /proc/sys/net/core/somaxconn. Default: 128
10:57:14.201 DEBUG [DefaultChannelId.<clinit>] - -Dio.netty.machineId: ce:f0:21:ff:fe:4d:c1:18 (auto-detected)
10:57:14.210 DEBUG [ResourceLeakDetector.<clinit>] - -Dio.netty.leakDetection.level: simple
10:57:14.210 DEBUG [ResourceLeakDetector.<clinit>] - -Dio.netty.leakDetection.targetRecords: 4
10:57:14.235 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.numHeapArenas: 24
10:57:14.235 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.numDirectArenas: 24
10:57:14.235 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.pageSize: 8192
10:57:14.235 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.maxOrder: 11
10:57:14.235 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.chunkSize: 16777216
10:57:14.235 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.smallCacheSize: 256
10:57:14.236 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.normalCacheSize: 64
10:57:14.236 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.maxCachedBufferCapacity: 32768
10:57:14.236 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.cacheTrimInterval: 8192
10:57:14.236 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.cacheTrimIntervalMillis: 0
10:57:14.236 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.useCacheForAllThreads: true
10:57:14.236 DEBUG [PooledByteBufAllocator.<clinit>] - -Dio.netty.allocator.maxCachedByteBuffersPerChunk: 1023
10:57:14.245 DEBUG [ByteBufUtil.<clinit>] - -Dio.netty.allocator.type: pooled
10:57:14.246 DEBUG [ByteBufUtil.<clinit>] - -Dio.netty.threadLocalDirectBufferSize: 0
10:57:14.246 DEBUG [ByteBufUtil.<clinit>] - -Dio.netty.maxThreadLocalCharBufferSize: 16384
10:57:14.259 DEBUG [LoggingHandler.channelRegistered] - [id: 0xe1a21335] REGISTERED
10:57:14.261 DEBUG [LoggingHandler.bind] - [id: 0xe1a21335] BIND: selgrid/10.200.96.126:55s01
java.lang.reflect.InvocationTargetException10:57:14.263 DEBUG [LoggingHandler.close] - [id: 0xe1a21335] CLOSE

	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.openqa.selenium.grid.Bootstrap.runMain(Bootstrap.java:77)
	at org.openqa.selenium.grid.Bootstrap.main(Bootstrap.java:70)
Caused by: java.io.UncheckedIOException: java.io.IOException: Port 5501 already in use
	at org.openqa.selenium.netty.server.NettyServer.start(NettyServer.java:159)
	at org.openqa.selenium.grid.node.httpd.NodeServer$1.start(NodeServer.java:192)
	at org.openqa.selenium.grid.node.httpd.NodeServer$1.start(NodeServer.java:189)
	at org.openqa.selenium.grid.node.httpd.NodeServer.execute(NodeServer.java:231)
	at org.openqa.selenium.grid.TemplateGridCommand.lambda$configure$4(TemplateGridCommand.java:129)
	at org.openqa.selenium.grid.Main.launch(Main.java:83)
	at org.openqa.selenium.grid.Main.go(Main.java:57)
	at org.openqa.selenium.grid.Main.main(Main.java:42)
	... 6 more
Caused by: java.io.IOException: Port 5501 already in use
	... 14 more
Caused by: java.net.BindException: Cannot assign requested address
	at java.base/sun.nio.ch.Net.bind0(Native Method)
	at java.base/sun.nio.ch.Net.bind(Net.java:455)
	at java.base/sun.nio.ch.Net.bind(Net.java:447)
	at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
10:57:14.264 DEBUG [LoggingHandler.channelUnregistered] - [id: 0xe1a21335] UNREGISTERED
	at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:134)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:562)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334)
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506)
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491)
	at io.netty.handler.logging.LoggingHandler.bind(LoggingHandler.java:230)
	at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506)
	at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491)
	at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973)
	at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:260)
	at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:356)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
2021-11-18 10:57:14,596 INFO exited: selenium-node (exit status 1; not expected)
```

I see the same thing with 4.1.0-20211123
Starting hub & nodes on separate VMs. Node start command is:
```
dudo docker run -v /dev/shm:/dev/shm --restart=always -d --name selenium-chrome -p 5555:5555 -p 5900:5900 -p 7900:7900 -e SE_NODE_HOST=$(curl http://169.254.169.254/latest/meta-data/local-ipv4) -e SE_EVENT_BUS_HOST=10.159.82.0 -e SE_EVENT_BUS_PUBLISH_PORT=4442 -e SE_EVENT_BUS_SUBSCRIBE_PORT=4443 -e LANG=en_GB.UTF-8 -e LANGUAGE=en_GB.UTF-8 -v /tmp/testfiles:/tmp/testfiles selenium/node-chrome:4.1.0-20211123
```
It can't bind on 5555. Container log is:
```
2021-11-26 12:13:23,396 INFO Included extra file "/etc/supervisor/conf.d/selenium.conf" during parsing
2021-11-26 12:13:23,400 INFO supervisord started with pid 8
2021-11-26 12:13:24,404 INFO spawned: 'xvfb' with pid 10
2021-11-26 12:13:24,406 INFO spawned: 'vnc' with pid 11
2021-11-26 12:13:24,407 INFO spawned: 'novnc' with pid 12
2021-11-26 12:13:24,409 INFO spawned: 'selenium-node' with pid 13
2021-11-26 12:13:25,411 INFO success: xvfb entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
2021-11-26 12:13:25,411 INFO success: vnc entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
2021-11-26 12:13:25,411 INFO success: novnc entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
2021-11-26 12:13:25,411 INFO success: selenium-node entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
Generating Selenium Config
Setting up SE_NODE_PORT...
Setting up SE_NODE_GRID_URL...
Selenium Grid Node configuration:
[events]
publish = "tcp://10.159.82.0:4442"
subscribe = "tcp://10.159.82.0:4443"

[server]
host = "10.159.78.239"
[node]
session-timeout = "300"
override-max-sessions = false
detect-drivers = false
max-sessions = 1

[[node.driver-configuration]]
display-name = "chrome"
stereotype = '{"browserName": "chrome", "browserVersion": "96.0", "platformName": "Linux"}'
max-sessions = 1

Starting Selenium Grid Node...
12:13:45.469 INFO [LoggingOptions.configureLogEncoding] - Using the system default encoding
12:13:45.481 INFO [OpenTelemetryTracer.createTracer] - Using OpenTelemetry for tracing
12:13:45.875 INFO [UnboundZmqEventBus.<init>] - Connecting to tcp://10.159.82.0:4442 and tcp://10.159.82.0:4443
12:13:45.973 INFO [UnboundZmqEventBus.<init>] - Sockets created
12:13:46.977 INFO [UnboundZmqEventBus.<init>] - Event bus ready
12:13:47.441 INFO [NodeServer.createHandlers] - Reporting self as: http://10.159.78.239:5555
12:13:47.493 INFO [NodeOptions.getSessionFactories] - Detected 2 available processors
12:13:47.575 INFO [NodeOptions.report] - Adding chrome for {"browserVersion": "96.0","browserName": "chrome","platformName": "Linux","se:vncEnabled": true} 1 times
12:13:47.589 INFO [Node.<init>] - Binding additional locator mechanisms: id, relative, namejava.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.openqa.selenium.grid.Bootstrap.runMain(Bootstrap.java:77)
        at org.openqa.selenium.grid.Bootstrap.main(Bootstrap.java:70)
Caused by: java.io.UncheckedIOException: java.io.IOException: Port 5555 already in use
        at org.openqa.selenium.netty.server.NettyServer.start(NettyServer.java:159)
        at org.openqa.selenium.grid.node.httpd.NodeServer$1.start(NodeServer.java:191)
        at org.openqa.selenium.grid.node.httpd.NodeServer$1.start(NodeServer.java:188)
        at org.openqa.selenium.grid.node.httpd.NodeServer.execute(NodeServer.java:230)
        at org.openqa.selenium.grid.TemplateGridCommand.lambda$configure$4(TemplateGridCommand.java:129)
        at org.openqa.selenium.grid.Main.launch(Main.java:83)
        at org.openqa.selenium.grid.Main.go(Main.java:57)
        at org.openqa.selenium.grid.Main.main(Main.java:42)
        ... 6 more
Caused by: java.io.IOException: Port 5555 already in use
        ... 14 more
Caused by: java.net.BindException: Cannot assign requested address
        at java.base/sun.nio.ch.Net.bind0(Native Method)
        at java.base/sun.nio.ch.Net.bind(Net.java:455)
        at java.base/sun.nio.ch.Net.bind(Net.java:447)
        at java.base/sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:227)
        at io.netty.channel.socket.nio.NioServerSocketChannel.doBind(NioServerSocketChannel.java:134)
        at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:562)
        at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1334)
        at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506)
        at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491)
        at io.netty.handler.logging.LoggingHandler.bind(LoggingHandler.java:230)
        at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:506)
        at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:491)
        at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:973)
        at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:260)
        at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:356)
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:829)
2021-11-26 12:13:48,530 INFO exited: selenium-node (exit status 1; not expected)
```

If I use `4.0.0-20211102` it works fine

--- a/Base/Dockerfile
+++ b/Base/Dockerfile
@@ -71,7 +71,7 @@ COPY supervisord.conf /etc
 RUN  mkdir -p /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
   && touch /opt/selenium/config.toml \
   && chmod -R 777 /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor /etc/passwd \
-  && wget --no-verbose https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.1.0/selenium-server-4.1.0.jar \
+  && wget --no-verbose https://github.com/SeleniumHQ/docker-selenium/releases/download/4.0.0-20211102/selenium_server_deploy.jar \
     -O /opt/selenium/selenium-server.jar \
   && chgrp -R 0 /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
   && chmod -R g=u /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor
@@ -80,6 +80,10 @@ RUN  mkdir -p /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/su
 # Run the following commands as non-privileged user
 #===================================================
 USER 1200:1201
+
+# Boolean value, maps "--bind-host"
+ENV SE_BIND_HOST false
+
 
 
 CMD ["/opt/bin/entry_point.sh"]

--- a/Distributor/start-selenium-grid-distributor.sh
+++ b/Distributor/start-selenium-grid-distributor.sh
@@ -60,6 +60,7 @@ java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar distributor \
   --sessionqueue-host "${SE_SESSION_QUEUE_HOST}" --sessionqueue-port "${SE_SESSION_QUEUE_PORT}" \
   --publish-events tcp://"${SE_EVENT_BUS_HOST}":"${SE_EVENT_BUS_PUBLISH_PORT}" \
   --subscribe-events tcp://"${SE_EVENT_BUS_HOST}":"${SE_EVENT_BUS_SUBSCRIBE_PORT}" \
+  --bind-host ${SE_BIND_HOST} \
   --bind-bus false \
   ${HOST_CONFIG} \
   ${PORT_CONFIG} \

--- a/EventBus/start-selenium-grid-eventbus.sh
+++ b/EventBus/start-selenium-grid-eventbus.sh
@@ -20,6 +20,7 @@ if [ ! -z "$SE_OPTS" ]; then
 fi
 
 java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar event-bus \
+  --bind-host ${SE_BIND_HOST} \
   ${HOST_CONFIG} \
   ${PORT_CONFIG} \
   ${SE_OPTS}

--- a/Hub/start-selenium-grid-hub.sh
+++ b/Hub/start-selenium-grid-hub.sh
@@ -22,6 +22,7 @@ java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar hub \
   --session-request-timeout ${SE_SESSION_REQUEST_TIMEOUT} \
   --session-retry-interval ${SE_SESSION_RETRY_INTERVAL} \
   --relax-checks ${SE_RELAX_CHECKS} \
+  --bind-host ${SE_BIND_HOST} \
   ${HOST_CONFIG} \
   ${PORT_CONFIG} \
   ${SE_OPTS}

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 NAME := $(or $(NAME),$(NAME),selenium)
 CURRENT_DATE := $(shell date '+%Y%m%d')
 BUILD_DATE := $(or $(BUILD_DATE),$(BUILD_DATE),$(CURRENT_DATE))
-VERSION := $(or $(VERSION),$(VERSION),4.0.0-beta-3)
+VERSION := $(or $(VERSION),$(VERSION),4.1.0)
 TAG_VERSION := $(VERSION)-$(BUILD_DATE)
 NAMESPACE := $(or $(NAMESPACE),$(NAMESPACE),$(NAME))
 AUTHORS := $(or $(AUTHORS),$(AUTHORS),SeleniumHQ)

--- a/NodeBase/start-selenium-node.sh
+++ b/NodeBase/start-selenium-node.sh
@@ -39,5 +39,6 @@ echo "Selenium Grid Node configuration: "
 cat "$CONFIG_FILE"
 echo "Starting Selenium Grid Node..."
 java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar node \
+  --bind-host ${SE_BIND_HOST} \
   --config "$CONFIG_FILE" \
   ${SE_OPTS}

--- a/Router/start-selenium-grid-router.sh
+++ b/Router/start-selenium-grid-router.sh
@@ -56,6 +56,7 @@ java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar router \
   --session-request-timeout ${SE_SESSION_REQUEST_TIMEOUT} \
   --session-retry-interval ${SE_SESSION_RETRY_INTERVAL} \
   --relax-checks true \
+  --bind-host ${SE_BIND_HOST} \
   ${HOST_CONFIG} \
   ${PORT_CONFIG} \
   ${SE_OPTS}

--- a/SessionQueue/start-selenium-grid-session-queue.sh
+++ b/SessionQueue/start-selenium-grid-session-queue.sh
@@ -39,6 +39,7 @@ java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar sessionqueue \
   --subscribe-events tcp://"${SE_EVENT_BUS_HOST}":${SE_EVENT_BUS_SUBSCRIBE_PORT} \
   --session-request-timeout ${SE_SESSION_REQUEST_TIMEOUT} \
   --session-retry-interval ${SE_SESSION_RETRY_INTERVAL} \
+  --bind-host ${SE_BIND_HOST} \
   ${HOST_CONFIG} \
   ${PORT_CONFIG} \
   ${SE_OPTS}

--- a/Sessions/start-selenium-grid-sessions.sh
+++ b/Sessions/start-selenium-grid-sessions.sh
@@ -37,6 +37,7 @@ fi
 java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar sessions \
   --publish-events tcp://"${SE_EVENT_BUS_HOST}":${SE_EVENT_BUS_PUBLISH_PORT} \
   --subscribe-events tcp://"${SE_EVENT_BUS_HOST}":${SE_EVENT_BUS_SUBSCRIBE_PORT} \
+  --bind-host ${SE_BIND_HOST} \
   ${HOST_CONFIG} \
   ${PORT_CONFIG} \
   ${SE_OPTS}

--- a/Standalone/start-selenium-standalone.sh
+++ b/Standalone/start-selenium-standalone.sh
@@ -12,5 +12,6 @@ echo "Selenium Grid Standalone configuration: "
 cat /opt/selenium/config.toml
 echo "Starting Selenium Grid Standalone..."
 java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar standalone \
+  --bind-host ${SE_BIND_HOST} \
   --config /opt/selenium/config.toml \
   ${SE_OPTS}

--- a/StandaloneDocker/start-selenium-grid-standalone-docker.sh
+++ b/StandaloneDocker/start-selenium-grid-standalone-docker.sh
@@ -17,5 +17,6 @@ fi
 java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar standalone \
   --relax-checks ${SE_RELAX_CHECKS} \
   --detect-drivers false \
+  --bind-host ${SE_BIND_HOST} \
   --config /opt/bin/config.toml \
   ${SE_GRID_URL} ${SE_OPTS}

